Makefile

```diff
@@ -19,6 +39,19 @@ push-ubi8-rust-builder: NAME = ubi8-rust-builder
 push-ubi8-rust-builder : build-ubi8-rust-builder login
 	$(call push,${NAME})
 
+pull-ubi8-rust-builder: NAME = ubi8-rust-builder
+pull-ubi8-rust-builder: login
+	$(call pull-arm64,${NAME})
+	$(call pull-amd64,${NAME})
+
+build-manifest-list: NAME = ubi8-rust-builder
+build-manifest-list:
+	$(call manifest,${NAME})
+
+push-manifest-list: NAME = ubi8-rust-builder
+push-manifest-list: login pull-ubi8-rust-builder build-manifest-list 
+	$(call manifest_push,${NAME})
```

Comment:
I assume this is meant to deduplicate `NAME`, yet we have to list it separately for each target anyway?

Reply:
I don't get it.... The targets should be generated via `latest-{ARCH}` but probably I misunderstand the issue.

Reply:
Removed pull, unnecessary. Hardcoded arch's wherever possible

Reply:
Is there a reason we don't define `NAME` on the top level, like `REPO` and co?

Reply:
not really, I just copied the other steps. i can do that :)